README.rst

```diff
@@ -27,17 +27,15 @@ Documentation
 
 See the READMEs for the early example applications:
 
-`ASRC Demo <https://github.com/xmos/sln_voice/blob/develop/examples/asrc_demo/README.rst>`_
+  * `ASRC Demo <https://github.com/xmos/sln_voice/blob/develop/examples/asrc_demo/README.rst>`_: an example design for an asynchronous sampling rate converter (ASRC)
 
-`FFD <https://github.com/xmos/sln_voice/blob/develop/examples/ffd/README.rst>`_
+  * `FFD <https://github.com/xmos/sln_voice/blob/develop/examples/ffd/README.rst>`_: two example designs for a far-field voice local control, each example contains a boot image and data partition binary
 
-`FFVA <https://github.com/xmos/sln_voice/blob/develop/examples/ffva/README.rst>`_
+  * `FFVA <https://github.com/xmos/sln_voice/blob/develop/examples/ffva/README.rst>`_: two example designs for a far-field voice assistant example design, each example contains a boot image and data partition binary
```

Comment:
This description says, `example designs ... example design`.  I suggest removing the 2nd phrase.